examples/charts/timeline-charts-example/README.md

```diff
@@ -41,11 +41,11 @@ _The following steps presume the use of npm, though yarn works as well._
 
 ## Preparing your Chart for Embedding
 
-This sample is preconfigured to render specific charts. You can run the sample as-is, or you can modify it to render your own charts by completing the following steps:
+This sample is preconfigured to render a specific chart. You can run the sample as-is, or you can modify it to render your own chart by completing the following steps:
```

Comment:
This was in plural intentionally as there are two charts in this example. :)

Reply:
As discussed we won't update this as it only applies to the one example :D